components/gitpod-db/src/typeorm/migration/1647333804782-IndexWorkspaceIdDeleted.ts

```diff
@@ -0,0 +1,22 @@
+/**
+ * Copyright (c) 2022 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License-AGPL.txt in the project root for license information.
+ */
+
+import { MigrationInterface, QueryRunner } from "typeorm";
+import { indexExists } from "./helper/helper";
+
+export const TABLE_NAME = "d_b_workspace";
```

Comment:
[nit] One could accidentally import just `TABLE_NAME` and lose the context that it relates to this migration (aside from the import) but I guess that's just TS for me. No action needed.